docs/changelog.rst

```diff
@@ -38,6 +38,8 @@ Other changes:
   (`gh-257 <https://github.com/django/django-localflavor/pull/257>`_). Users will need to generate migrations for any
   model fields they use with 'makemigrations'.
 
+- Adapting django deprecation policy
```

Comment:
This doesn't mean much to me. If you included the roadmap in the docs, you could link to it.

Also, include details of the deprecation in the docs.